test/e2e/tests/add-account.spec.js

```diff
@@ -36,4 +44,216 @@ describe('Add account', function () {
       },
     );
   });
+
```

Comment:
test description looks confusing `should add the same account addresses if added and then the same mnemonic is imported`

Reply:
maybe something like: `'should add the same account addresses when a secret recovery phrase is imported, the account is locked, and the same secret recovery phrase is imported again'`?